main_flow/main_flow.py

```diff
@@ -9,10 +10,10 @@
 import data_loader
 from training import training_test_cv_divider
 from utils import presistor
-from utils.logger import create_logger_instance
 from stemming import stemming
 
-main_flow_logger = create_logger_instance('Main_Flow')
+logging.basicConfig(level=logging.DEBUG, format='%(asctime)s:%(levelname)s:%(message)s')
+logger = logging.getLogger('main_flow')
```

Comment:
Is there a reason behind setting up the logging config in this file or it was like that before?

Reply:
It was like this before